### ISSUE TITLE
Possible bug with vector point_data?

### ISSUE THREAD

**Issue by mountaindust** [pyvista/pyvista-support#438]
## Description

I'm a bit of a VTK newbie, so I didn't want to post this for sure as a bug. But there's something strange going on with assigning point data. The [pyvista sphere example](https://docs.pyvista.org/examples/01-filter/glyphs.html) provides a minimal working example of what happens, so no data is needed:
'''python
sphere = pv.Sphere(radius=3.14)

# make cool swirly pattern
vectors = np.vstack(
    (
        np.sin(sphere.points[:, 0]),
        np.cos(sphere.points[:, 1]),
        np.cos(sphere.points[:, 2]),
    )
).T

# add and scale
sphere.vectors = vectors * 0.3

# plot just the arrows
sphere.arrows.plot(scalars='GlyphScale')
'''
All of this, of course, works fine. However, suppose you try to save this data in ascii format:
'''python
sphere.save('sphere.vtk', binary=False)
'''
If you open the resulting file in a text editor and scroll down to the POINT_DATA field, you get something that to me was rather unexpected:

    POINT_DATA 842
    SCALARS _vectors float 3
    LOOKUP_TABLE default

So, it's recording the vector data as scalar data with the name "_vectors", type float, and 3 components. This to me is rather weird. I was expecting a VECTOR dataset attribute with some kind of default name (I'm not sure how one assigns a name to vector data in the pyvista API). Something like:

    POINT_DATA 842
    VECTORS vectors float

I'm decently sure this is a bug because causes problems down the line: if I try to open this vtk file in something like VisIt to do a vector glyph plot, it can't find the vector data. But maybe there's some other way you are supposed to accomplish assigning vector point data? I thought the sphere example was a great showcase of how this should be done, but then there's this weird formatting issue. Any help/guidance would be greatly appreciated!

**Comment by davidcortesortuno** [comment]
Hi, I was going to raise an issue that might be related. You mentioned that to add `vector` data you can do `sphere.vectors = vectors * 0.3`, but I'm not sure if there is a better method to do this, because you cannot specify a `name` with this approach. In the documentation I found a method called `add_field_array` but this creates a `Fields` field, where it might be better to have a `Points` or `Cells` field. Maybe in this way you can have a proper vector object.

**Comment by mountaindust** [comment]
@davidcortesortuno Right. I actually meant to refer to this point in my original issue, but apparently I forgot. With `add_points_array`, it's setup like a dictionary so that you can add a key-value pair and thus get a name for the data. But `vectors` does not appear to be set up like this. It probably should be though.

I'm pretty convinced this is a bug now, so it may be worth migrating the issue to the main issue tracker of the repository where it will likely get more attention.

**Comment by davidcortesortuno** [comment]
Yes, from what I understand in the code, it seems that when saving the data the `_vectors` data might be set with the `SCALARS` attribute. I think this can be specified via `vtk` using the `SetAttribute()` method https://vtk.org/doc/nightly/html/classvtkDataSetAttributes.html#details However I could not find where attributes are set in pyvista

**Comment by MatthewFlamm** [comment]
> because you cannot specify a `name` with this approach

See #448 on how to set a name.  This is for others who may see this issue first.

**Comment by MatthewFlamm** [comment]
> Yes, from what I understand in the code, it seems that when saving the data the `_vectors` data might be set with the `SCALARS` attribute. I think this can be specified via `vtk` using the `SetAttribute()` method https://vtk.org/doc/nightly/html/classvtkDataSetAttributes.html#details However I could not find where attributes are set in pyvista

Here is the dataset attributes class in pyvista, I do not see any usage of setting the type of attribute currently.
https://github.com/pyvista/pyvista-support/issues/438
